ledger-service/lf-value-json/src/main/scala/com/digitalasset/daml/lf/value/json/ApiCodecCompressed.scala

```diff
@@ -33,21 +34,21 @@ object ApiCodecCompressed {
   def apiValueToJsValue(value: Model.ApiValue): JsValue = value match {
     case v: Model.ApiRecord => apiRecordToJsValue(v)
```

Comment:
I like that you got rid of the `Model.X` aliases. But I think you missed a few cases, like this one above.

Reply:
The cases I skipped were deliberate, as replacing them requires actual thinking.